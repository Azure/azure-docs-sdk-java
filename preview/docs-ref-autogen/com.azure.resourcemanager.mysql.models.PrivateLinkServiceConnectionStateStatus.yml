### YamlMime:JavaType
uid: "com.azure.resourcemanager.mysql.models.PrivateLinkServiceConnectionStateStatus"
fullName: "com.azure.resourcemanager.mysql.models.PrivateLinkServiceConnectionStateStatus"
name: "PrivateLinkServiceConnectionStateStatus"
nameWithType: "PrivateLinkServiceConnectionStateStatus"
summary: "Defines values for PrivateLinkServiceConnectionStateStatus."
inheritances:
- "<xref href=\"java.lang.Object\" data-throw-if-not-resolved=\"False\" />"
- "<xref href=\"com.azure.core.util.ExpandableStringEnum\" data-throw-if-not-resolved=\"False\" />"
inheritedMembers:
- "com.azure.core.util.ExpandableStringEnum.<T>fromString(java.lang.String,java.lang.Class<T>)"
- "com.azure.core.util.ExpandableStringEnum.<T>values(java.lang.Class<T>)"
- "com.azure.core.util.ExpandableStringEnum.equals(java.lang.Object)"
- "com.azure.core.util.ExpandableStringEnum.hashCode()"
- "com.azure.core.util.ExpandableStringEnum.toString()"
- "java.lang.Object.clone()"
- "java.lang.Object.finalize()"
- "java.lang.Object.getClass()"
- "java.lang.Object.notify()"
- "java.lang.Object.notifyAll()"
- "java.lang.Object.wait()"
- "java.lang.Object.wait(long)"
- "java.lang.Object.wait(long,int)"
syntax: "public final class PrivateLinkServiceConnectionStateStatus extends ExpandableStringEnum<PrivateLinkServiceConnectionStateStatus>"
constructors:
- "com.azure.resourcemanager.mysql.models.PrivateLinkServiceConnectionStateStatus.PrivateLinkServiceConnectionStateStatus()"
fields:
- "com.azure.resourcemanager.mysql.models.PrivateLinkServiceConnectionStateStatus.APPROVED"
- "com.azure.resourcemanager.mysql.models.PrivateLinkServiceConnectionStateStatus.DISCONNECTED"
- "com.azure.resourcemanager.mysql.models.PrivateLinkServiceConnectionStateStatus.PENDING"
- "com.azure.resourcemanager.mysql.models.PrivateLinkServiceConnectionStateStatus.REJECTED"
methods:
- "com.azure.resourcemanager.mysql.models.PrivateLinkServiceConnectionStateStatus.fromString(java.lang.String)"
- "com.azure.resourcemanager.mysql.models.PrivateLinkServiceConnectionStateStatus.values()"
type: "class"
metadata: {}
package: "com.azure.resourcemanager.mysql.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-mysql:1.0.0-beta.1
