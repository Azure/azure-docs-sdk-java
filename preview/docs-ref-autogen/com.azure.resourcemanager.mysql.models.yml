### YamlMime:JavaPackage
uid: "com.azure.resourcemanager.mysql.models"
fullName: "com.azure.resourcemanager.mysql.models"
name: "com.azure.resourcemanager.mysql.models"
summary: "Package containing the data models for MySqlManagementClient. The Microsoft Azure management API provides create,\n read, update, and delete functionality for Azure MySQL resources including servers, databases, firewall rules, VNET\n rules, log files and configurations with new business model."
classes:
- "com.azure.resourcemanager.mysql.models.AdvisorsResultList"
- "com.azure.resourcemanager.mysql.models.CreateMode"
- "com.azure.resourcemanager.mysql.models.DatabaseListResult"
- "com.azure.resourcemanager.mysql.models.ErrorAdditionalInfo"
- "com.azure.resourcemanager.mysql.models.FirewallRuleListResult"
- "com.azure.resourcemanager.mysql.models.GeoRedundantBackup"
- "com.azure.resourcemanager.mysql.models.IdentityType"
- "com.azure.resourcemanager.mysql.models.InfrastructureEncryption"
- "com.azure.resourcemanager.mysql.models.LogFileListResult"
- "com.azure.resourcemanager.mysql.models.MinimalTlsVersionEnum"
- "com.azure.resourcemanager.mysql.models.NameAvailabilityRequest"
- "com.azure.resourcemanager.mysql.models.Operation"
- "com.azure.resourcemanager.mysql.models.OperationDisplay"
- "com.azure.resourcemanager.mysql.models.OperationOrigin"
- "com.azure.resourcemanager.mysql.models.PerformanceTierListResult"
- "com.azure.resourcemanager.mysql.models.PerformanceTierServiceLevelObjectives"
- "com.azure.resourcemanager.mysql.models.PrivateEndpointConnectionListResult"
- "com.azure.resourcemanager.mysql.models.PrivateEndpointProperty"
- "com.azure.resourcemanager.mysql.models.PrivateEndpointProvisioningState"
- "com.azure.resourcemanager.mysql.models.PrivateLinkResourceListResult"
- "com.azure.resourcemanager.mysql.models.PrivateLinkResourceProperties"
- "com.azure.resourcemanager.mysql.models.PrivateLinkServiceConnectionStateActionsRequire"
- "com.azure.resourcemanager.mysql.models.PrivateLinkServiceConnectionStateProperty"
- "com.azure.resourcemanager.mysql.models.PrivateLinkServiceConnectionStateStatus"
- "com.azure.resourcemanager.mysql.models.PublicNetworkAccessEnum"
- "com.azure.resourcemanager.mysql.models.QueryPerformanceInsightResetDataResultState"
- "com.azure.resourcemanager.mysql.models.QueryTextsResultList"
- "com.azure.resourcemanager.mysql.models.RecommendationActionsResultList"
- "com.azure.resourcemanager.mysql.models.ResourceIdentity"
- "com.azure.resourcemanager.mysql.models.SecurityAlertPolicyName"
- "com.azure.resourcemanager.mysql.models.ServerAdministratorResourceListResult"
- "com.azure.resourcemanager.mysql.models.ServerForCreate"
- "com.azure.resourcemanager.mysql.models.ServerKeyListResult"
- "com.azure.resourcemanager.mysql.models.ServerKeyType"
- "com.azure.resourcemanager.mysql.models.ServerListResult"
- "com.azure.resourcemanager.mysql.models.ServerPrivateEndpointConnection"
- "com.azure.resourcemanager.mysql.models.ServerPrivateEndpointConnectionProperties"
- "com.azure.resourcemanager.mysql.models.ServerPrivateLinkServiceConnectionStateProperty"
- "com.azure.resourcemanager.mysql.models.ServerPropertiesForCreate"
- "com.azure.resourcemanager.mysql.models.ServerPropertiesForDefaultCreate"
- "com.azure.resourcemanager.mysql.models.ServerPropertiesForGeoRestore"
- "com.azure.resourcemanager.mysql.models.ServerPropertiesForReplica"
- "com.azure.resourcemanager.mysql.models.ServerPropertiesForRestore"
- "com.azure.resourcemanager.mysql.models.ServerState"
- "com.azure.resourcemanager.mysql.models.ServerUpdateParameters"
- "com.azure.resourcemanager.mysql.models.ServerUpgradeParameters"
- "com.azure.resourcemanager.mysql.models.ServerVersion"
- "com.azure.resourcemanager.mysql.models.Sku"
- "com.azure.resourcemanager.mysql.models.SkuTier"
- "com.azure.resourcemanager.mysql.models.StorageAutogrow"
- "com.azure.resourcemanager.mysql.models.StorageProfile"
- "com.azure.resourcemanager.mysql.models.TagsObject"
- "com.azure.resourcemanager.mysql.models.TopQueryStatisticsInput"
- "com.azure.resourcemanager.mysql.models.TopQueryStatisticsResultList"
- "com.azure.resourcemanager.mysql.models.VirtualNetworkRuleListResult"
- "com.azure.resourcemanager.mysql.models.VirtualNetworkRuleState"
- "com.azure.resourcemanager.mysql.models.WaitStatisticsInput"
- "com.azure.resourcemanager.mysql.models.WaitStatisticsResultList"
enums:
- "com.azure.resourcemanager.mysql.models.AdministratorType"
- "com.azure.resourcemanager.mysql.models.ServerSecurityAlertPolicyState"
- "com.azure.resourcemanager.mysql.models.SslEnforcementEnum"
interfaces:
- "com.azure.resourcemanager.mysql.models.Advisor"
- "com.azure.resourcemanager.mysql.models.Advisors"
- "com.azure.resourcemanager.mysql.models.CheckNameAvailabilities"
- "com.azure.resourcemanager.mysql.models.Configuration"
- "com.azure.resourcemanager.mysql.models.Configuration.Definition"
- "com.azure.resourcemanager.mysql.models.Configuration.DefinitionStages"
- "com.azure.resourcemanager.mysql.models.Configuration.DefinitionStages.Blank"
- "com.azure.resourcemanager.mysql.models.Configuration.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.mysql.models.Configuration.DefinitionStages.WithParentResource"
- "com.azure.resourcemanager.mysql.models.Configuration.DefinitionStages.WithSource"
- "com.azure.resourcemanager.mysql.models.Configuration.DefinitionStages.WithValue"
- "com.azure.resourcemanager.mysql.models.Configuration.Update"
- "com.azure.resourcemanager.mysql.models.Configuration.UpdateStages"
- "com.azure.resourcemanager.mysql.models.Configuration.UpdateStages.WithSource"
- "com.azure.resourcemanager.mysql.models.Configuration.UpdateStages.WithValue"
- "com.azure.resourcemanager.mysql.models.ConfigurationListResult"
- "com.azure.resourcemanager.mysql.models.Configurations"
- "com.azure.resourcemanager.mysql.models.Database"
- "com.azure.resourcemanager.mysql.models.Database.Definition"
- "com.azure.resourcemanager.mysql.models.Database.DefinitionStages"
- "com.azure.resourcemanager.mysql.models.Database.DefinitionStages.Blank"
- "com.azure.resourcemanager.mysql.models.Database.DefinitionStages.WithCharset"
- "com.azure.resourcemanager.mysql.models.Database.DefinitionStages.WithCollation"
- "com.azure.resourcemanager.mysql.models.Database.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.mysql.models.Database.DefinitionStages.WithParentResource"
- "com.azure.resourcemanager.mysql.models.Database.Update"
- "com.azure.resourcemanager.mysql.models.Database.UpdateStages"
- "com.azure.resourcemanager.mysql.models.Database.UpdateStages.WithCharset"
- "com.azure.resourcemanager.mysql.models.Database.UpdateStages.WithCollation"
- "com.azure.resourcemanager.mysql.models.Databases"
- "com.azure.resourcemanager.mysql.models.FirewallRule"
- "com.azure.resourcemanager.mysql.models.FirewallRule.Definition"
- "com.azure.resourcemanager.mysql.models.FirewallRule.DefinitionStages"
- "com.azure.resourcemanager.mysql.models.FirewallRule.DefinitionStages.Blank"
- "com.azure.resourcemanager.mysql.models.FirewallRule.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.mysql.models.FirewallRule.DefinitionStages.WithEndIpAddress"
- "com.azure.resourcemanager.mysql.models.FirewallRule.DefinitionStages.WithParentResource"
- "com.azure.resourcemanager.mysql.models.FirewallRule.DefinitionStages.WithStartIpAddress"
- "com.azure.resourcemanager.mysql.models.FirewallRule.Update"
- "com.azure.resourcemanager.mysql.models.FirewallRule.UpdateStages"
- "com.azure.resourcemanager.mysql.models.FirewallRule.UpdateStages.WithEndIpAddress"
- "com.azure.resourcemanager.mysql.models.FirewallRule.UpdateStages.WithStartIpAddress"
- "com.azure.resourcemanager.mysql.models.FirewallRules"
- "com.azure.resourcemanager.mysql.models.LocationBasedPerformanceTiers"
- "com.azure.resourcemanager.mysql.models.LocationBasedRecommendedActionSessionsOperationStatus"
- "com.azure.resourcemanager.mysql.models.LocationBasedRecommendedActionSessionsResults"
- "com.azure.resourcemanager.mysql.models.LogFile"
- "com.azure.resourcemanager.mysql.models.LogFiles"
- "com.azure.resourcemanager.mysql.models.NameAvailability"
- "com.azure.resourcemanager.mysql.models.OperationListResult"
- "com.azure.resourcemanager.mysql.models.Operations"
- "com.azure.resourcemanager.mysql.models.PerformanceTierProperties"
- "com.azure.resourcemanager.mysql.models.PrivateEndpointConnection"
- "com.azure.resourcemanager.mysql.models.PrivateEndpointConnection.Definition"
- "com.azure.resourcemanager.mysql.models.PrivateEndpointConnection.DefinitionStages"
- "com.azure.resourcemanager.mysql.models.PrivateEndpointConnection.DefinitionStages.Blank"
- "com.azure.resourcemanager.mysql.models.PrivateEndpointConnection.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.mysql.models.PrivateEndpointConnection.DefinitionStages.WithParentResource"
- "com.azure.resourcemanager.mysql.models.PrivateEndpointConnection.DefinitionStages.WithPrivateEndpoint"
- "com.azure.resourcemanager.mysql.models.PrivateEndpointConnection.DefinitionStages.WithPrivateLinkServiceConnectionState"
- "com.azure.resourcemanager.mysql.models.PrivateEndpointConnection.Update"
- "com.azure.resourcemanager.mysql.models.PrivateEndpointConnection.UpdateStages"
- "com.azure.resourcemanager.mysql.models.PrivateEndpointConnection.UpdateStages.WithTags"
- "com.azure.resourcemanager.mysql.models.PrivateEndpointConnections"
- "com.azure.resourcemanager.mysql.models.PrivateLinkResource"
- "com.azure.resourcemanager.mysql.models.PrivateLinkResources"
- "com.azure.resourcemanager.mysql.models.QueryPerformanceInsightResetDataResult"
- "com.azure.resourcemanager.mysql.models.QueryStatistic"
- "com.azure.resourcemanager.mysql.models.QueryText"
- "com.azure.resourcemanager.mysql.models.QueryTexts"
- "com.azure.resourcemanager.mysql.models.RecommendationAction"
- "com.azure.resourcemanager.mysql.models.RecommendedActionSessionsOperationStatus"
- "com.azure.resourcemanager.mysql.models.RecommendedActions"
- "com.azure.resourcemanager.mysql.models.RecoverableServerResource"
- "com.azure.resourcemanager.mysql.models.RecoverableServers"
- "com.azure.resourcemanager.mysql.models.Replicas"
- "com.azure.resourcemanager.mysql.models.ResourceProviders"
- "com.azure.resourcemanager.mysql.models.Server"
- "com.azure.resourcemanager.mysql.models.Server.Definition"
- "com.azure.resourcemanager.mysql.models.Server.DefinitionStages"
- "com.azure.resourcemanager.mysql.models.Server.DefinitionStages.Blank"
- "com.azure.resourcemanager.mysql.models.Server.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.mysql.models.Server.DefinitionStages.WithIdentity"
- "com.azure.resourcemanager.mysql.models.Server.DefinitionStages.WithLocation"
- "com.azure.resourcemanager.mysql.models.Server.DefinitionStages.WithProperties"
- "com.azure.resourcemanager.mysql.models.Server.DefinitionStages.WithResourceGroup"
- "com.azure.resourcemanager.mysql.models.Server.DefinitionStages.WithSku"
- "com.azure.resourcemanager.mysql.models.Server.DefinitionStages.WithTags"
- "com.azure.resourcemanager.mysql.models.Server.Update"
- "com.azure.resourcemanager.mysql.models.Server.UpdateStages"
- "com.azure.resourcemanager.mysql.models.Server.UpdateStages.WithAdministratorLoginPassword"
- "com.azure.resourcemanager.mysql.models.Server.UpdateStages.WithIdentity"
- "com.azure.resourcemanager.mysql.models.Server.UpdateStages.WithMinimalTlsVersion"
- "com.azure.resourcemanager.mysql.models.Server.UpdateStages.WithPublicNetworkAccess"
- "com.azure.resourcemanager.mysql.models.Server.UpdateStages.WithReplicationRole"
- "com.azure.resourcemanager.mysql.models.Server.UpdateStages.WithSku"
- "com.azure.resourcemanager.mysql.models.Server.UpdateStages.WithSslEnforcement"
- "com.azure.resourcemanager.mysql.models.Server.UpdateStages.WithStorageProfile"
- "com.azure.resourcemanager.mysql.models.Server.UpdateStages.WithTags"
- "com.azure.resourcemanager.mysql.models.Server.UpdateStages.WithVersion"
- "com.azure.resourcemanager.mysql.models.ServerAdministratorResource"
- "com.azure.resourcemanager.mysql.models.ServerAdministrators"
- "com.azure.resourcemanager.mysql.models.ServerBasedPerformanceTiers"
- "com.azure.resourcemanager.mysql.models.ServerKey"
- "com.azure.resourcemanager.mysql.models.ServerKey.Definition"
- "com.azure.resourcemanager.mysql.models.ServerKey.DefinitionStages"
- "com.azure.resourcemanager.mysql.models.ServerKey.DefinitionStages.Blank"
- "com.azure.resourcemanager.mysql.models.ServerKey.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.mysql.models.ServerKey.DefinitionStages.WithParentResource"
- "com.azure.resourcemanager.mysql.models.ServerKey.DefinitionStages.WithServerKeyType"
- "com.azure.resourcemanager.mysql.models.ServerKey.DefinitionStages.WithUri"
- "com.azure.resourcemanager.mysql.models.ServerKey.Update"
- "com.azure.resourcemanager.mysql.models.ServerKey.UpdateStages"
- "com.azure.resourcemanager.mysql.models.ServerKey.UpdateStages.WithServerKeyType"
- "com.azure.resourcemanager.mysql.models.ServerKey.UpdateStages.WithUri"
- "com.azure.resourcemanager.mysql.models.ServerKeys"
- "com.azure.resourcemanager.mysql.models.ServerParameters"
- "com.azure.resourcemanager.mysql.models.ServerSecurityAlertPolicies"
- "com.azure.resourcemanager.mysql.models.ServerSecurityAlertPolicy"
- "com.azure.resourcemanager.mysql.models.ServerSecurityAlertPolicy.Definition"
- "com.azure.resourcemanager.mysql.models.ServerSecurityAlertPolicy.DefinitionStages"
- "com.azure.resourcemanager.mysql.models.ServerSecurityAlertPolicy.DefinitionStages.Blank"
- "com.azure.resourcemanager.mysql.models.ServerSecurityAlertPolicy.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.mysql.models.ServerSecurityAlertPolicy.DefinitionStages.WithDisabledAlerts"
- "com.azure.resourcemanager.mysql.models.ServerSecurityAlertPolicy.DefinitionStages.WithEmailAccountAdmins"
- "com.azure.resourcemanager.mysql.models.ServerSecurityAlertPolicy.DefinitionStages.WithEmailAddresses"
- "com.azure.resourcemanager.mysql.models.ServerSecurityAlertPolicy.DefinitionStages.WithParentResource"
- "com.azure.resourcemanager.mysql.models.ServerSecurityAlertPolicy.DefinitionStages.WithRetentionDays"
- "com.azure.resourcemanager.mysql.models.ServerSecurityAlertPolicy.DefinitionStages.WithState"
- "com.azure.resourcemanager.mysql.models.ServerSecurityAlertPolicy.DefinitionStages.WithStorageAccountAccessKey"
- "com.azure.resourcemanager.mysql.models.ServerSecurityAlertPolicy.DefinitionStages.WithStorageEndpoint"
- "com.azure.resourcemanager.mysql.models.ServerSecurityAlertPolicy.Update"
- "com.azure.resourcemanager.mysql.models.ServerSecurityAlertPolicy.UpdateStages"
- "com.azure.resourcemanager.mysql.models.ServerSecurityAlertPolicy.UpdateStages.WithDisabledAlerts"
- "com.azure.resourcemanager.mysql.models.ServerSecurityAlertPolicy.UpdateStages.WithEmailAccountAdmins"
- "com.azure.resourcemanager.mysql.models.ServerSecurityAlertPolicy.UpdateStages.WithEmailAddresses"
- "com.azure.resourcemanager.mysql.models.ServerSecurityAlertPolicy.UpdateStages.WithRetentionDays"
- "com.azure.resourcemanager.mysql.models.ServerSecurityAlertPolicy.UpdateStages.WithState"
- "com.azure.resourcemanager.mysql.models.ServerSecurityAlertPolicy.UpdateStages.WithStorageAccountAccessKey"
- "com.azure.resourcemanager.mysql.models.ServerSecurityAlertPolicy.UpdateStages.WithStorageEndpoint"
- "com.azure.resourcemanager.mysql.models.Servers"
- "com.azure.resourcemanager.mysql.models.TopQueryStatistics"
- "com.azure.resourcemanager.mysql.models.VirtualNetworkRule"
- "com.azure.resourcemanager.mysql.models.VirtualNetworkRule.Definition"
- "com.azure.resourcemanager.mysql.models.VirtualNetworkRule.DefinitionStages"
- "com.azure.resourcemanager.mysql.models.VirtualNetworkRule.DefinitionStages.Blank"
- "com.azure.resourcemanager.mysql.models.VirtualNetworkRule.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.mysql.models.VirtualNetworkRule.DefinitionStages.WithIgnoreMissingVnetServiceEndpoint"
- "com.azure.resourcemanager.mysql.models.VirtualNetworkRule.DefinitionStages.WithParentResource"
- "com.azure.resourcemanager.mysql.models.VirtualNetworkRule.DefinitionStages.WithVirtualNetworkSubnetId"
- "com.azure.resourcemanager.mysql.models.VirtualNetworkRule.Update"
- "com.azure.resourcemanager.mysql.models.VirtualNetworkRule.UpdateStages"
- "com.azure.resourcemanager.mysql.models.VirtualNetworkRule.UpdateStages.WithIgnoreMissingVnetServiceEndpoint"
- "com.azure.resourcemanager.mysql.models.VirtualNetworkRule.UpdateStages.WithVirtualNetworkSubnetId"
- "com.azure.resourcemanager.mysql.models.VirtualNetworkRules"
- "com.azure.resourcemanager.mysql.models.WaitStatistic"
- "com.azure.resourcemanager.mysql.models.WaitStatistics"
metadata: {}
package: "com.azure.resourcemanager.mysql.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-mysql:1.0.0-beta.1
