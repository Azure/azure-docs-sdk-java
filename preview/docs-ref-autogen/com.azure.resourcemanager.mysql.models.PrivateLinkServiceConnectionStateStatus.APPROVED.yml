### YamlMime:JavaMember
uid: "com.azure.resourcemanager.mysql.models.PrivateLinkServiceConnectionStateStatus.APPROVED*"
fullName: "com.azure.resourcemanager.mysql.models.PrivateLinkServiceConnectionStateStatus.APPROVED"
name: "APPROVED"
nameWithType: "PrivateLinkServiceConnectionStateStatus.APPROVED"
members:
- uid: "com.azure.resourcemanager.mysql.models.PrivateLinkServiceConnectionStateStatus.APPROVED"
  fullName: "com.azure.resourcemanager.mysql.models.PrivateLinkServiceConnectionStateStatus.APPROVED"
  name: "APPROVED"
  nameWithType: "PrivateLinkServiceConnectionStateStatus.APPROVED"
  summary: "Static value Approved for PrivateLinkServiceConnectionStateStatus."
  syntax: "public static final PrivateLinkServiceConnectionStateStatus APPROVED"
type: "field"
metadata: {}
package: "com.azure.resourcemanager.mysql.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-mysql:1.0.0-beta.1
